os/fs/smartfs/smartfs_utils.c

```diff
@@ -1496,7 +1492,7 @@ int smartfs_invalidateentry(struct smartfs_mountpt_s *fs, uint16_t parentdirsect
 
 	entry_flags = (uint8_t *)&fs->fs_rwbuffer[0];
 #if CONFIG_SMARTFS_ERASEDSTATE == 0xFF
-#if CONFIG_SMARTFS_ALIGNED_ACCESS
+#ifdef CONFIG_SMARTFS_ALIGNED_ACCESS
```

Comment:
you can add this change separately under a different commit (refactoring)

Reply:
Ohkay